This patch fixes tk 8.6.13 error on MacOS.

Full details are here: https://core.tcl-lang.org/tk/tktview/8912083dc

diff -rubBaN tk.orig/macosx/tkMacOSXFont.c tk/macosx/tkMacOSXFont.c
--- tk.orig/macosx/tkMacOSXFont.c	2024-02-22 20:58:22.528455200 +0200
+++ tk/macosx/tkMacOSXFont.c	2024-02-22 20:58:45.591674800 +0200
@@ -171,10 +171,10 @@
     return _ds;
 }

-#ifndef __clang__
+//#ifndef __clang__
 @synthesize UTF8String = _UTF8String;
 @synthesize DString = _ds;
-#endif
+//#endif
 @end
 
 #define GetNSFontTraitsFromTkFontAttributes(faPtr) \
